README.md

```diff
@@ -48,16 +48,16 @@ After succesfully setting up your dev environment, navigate to [http://localhost
 ### Configure back-end connection
 To connect to front-end to your local back-end you will first need to set up your back-end, you can follow the installation guide provided [here](https://github.com/OpenCatalogi/OpenCatalogiBundle#opencatalogibundle-).
 
-To connect the app to your local gateway, you will need to change the environment variables in the `env.js` file. Replace the content of the file with the following:
+To connect the app to your local gateway, you will need to change the environment variables in the `.env.development` file. Replace the content below Backend-config with the following:
 ```
-window.sessionStorage.setItem("GATSBY_ME_URL", "http://localhost/api/users/me");
-window.sessionStorage.setItem("GATSBY_API_URL", "http://localhost/api");
-window.sessionStorage.setItem("GATSBY_ADMIN_URL", "http://localhost/admin");
-window.sessionStorage.setItem("GATSBY_BASE_URL", "http://localhost");
-window.sessionStorage.setItem("GATSBY_FRONTEND_URL", "http://localhost");
-window.sessionStorage.setItem("GATSBY_ORGANIZATION", "");
-window.sessionStorage.setItem("GATSBY_LOGIN_REDIRECT", "vault");
-window.sessionStorage.setItem("ADMIN_DASHBOARD_URL", "http://localhost:8000");
+GATSBY_ME_URL=http://localhost/me
+GATSBY_API_URL=http://localhost/api
+GATSBY_ADMIN_URL=http://localhost/admin
+GATSBY_BASE_URL=http://localhost
+GATSBY_FRONTEND_URL=http://localhost:8000
+GATSBY_ORGANIZATION=http://webresourcecatalogus.conduction.svc.cluster.local/organizations/b2d3176e-f1c6-4365-ab86-dd253c65fc43
+GATSBY_LOGIN_REDIRECT=vault
+GATSBY_ADMIN_DASHBOARD_URL=https://admin.opencatalogi.nl
```

Comment:
Shouldn't the `.env.development` come with this content out-of-the-box? Making the instruction to replace something redundant?